packages/text-field/test/text-field.test.js

```diff
@@ -26,49 +29,48 @@ describe('text-field', () => {
         }
       }
 
-      function assertPropCanBeSet(prop, value) {
-        for (let i = 0; i < 3; i++) {
-          // Check different values (i.e. true false true for boolean or string1 string2 string3)
-          const newValue = typeof value === 'boolean' ? i % 2 === 0 : value + i;
-          textField[prop] = newValue;
-          expect(input[prop]).to.be.equal(newValue);
-        }
-      }
-
       ['pattern', 'placeholder', 'value', 'title'].forEach((prop) => {
-        it(`should set string property ${prop}`, () => {
-          assertPropCanBeSet(prop, 'foo');
+        it(`should set string property ${prop}`, async () => {
```

Comment:
**note:** We could test some properties from the list with snapshot tests.

Reply:
Same as above, let's consider replacing these with snapshots in a separate PR.